src/components/Cards/Cards.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import themeGet from 'extensions/themeGet';
+import Card from './Card';
+
+const Cards = styled.div`
+  display: flex;
+
+  & > * {
+    margin: ${themeGet('spacing.medium')};
+    border-radius: 5px;
```

Comment:
I think `Card` by itself should apply border radius and shadow even if it's not enclosed in `Cards`.